### PR TITLE
Add LIDAR sensors

### PR DESCRIPTION
**Pull request by ecourtois**

### CODE REVIEW

File: cogip/sensor.py [hunk 1]
```diff
@@ -8,46 +9,61 @@
...
         self.distance = None
 
-        direction_x = 1.0 if origin_x > 0 else -1.0 if origin_x < 0 else 0.0
-        direction_y = 1.0 if origin_y > 0 else -1.0 if origin_y < 0 else 0.0
-
         self.ray_caster = Qt3DRender.QRayCaster()
-        self.ray_caster.setEnabled(True)
-        self.ray_caster.setLength(0)  # Infinite
+        self.ray_caster.setEnabled(False)
+        self.ray_caster.setLength(2000)  # Infinite
```
Comment (gdoffe) [inline cogip/sensor.py:41]: It seems no more infinite.
Why 2000 ? Can you create a constant to define this ?

File: cogip/sensor.py [hunk 2]
```diff
@@ -8,46 +9,61 @@
...
+            self.add_obstacle_layer(obstacle)
+
         # Add impact entity
         self.impact_entity = ImpactEntity(radius=impact_radius, color=impact_color)
         self.impact_entity.setParent(self.asset_entity)
 
         self.timer = QtCore.QTimer()
         self.timer.timeout.connect(self.handle_hits)
-        self.timer.start(50)
+        self.timer.start(100)
```
Comment (gdoffe) [inline cogip/sensor.py:59]: what is this timer for and why changing 50 to 100 ? Is this milliseconds ?

**Review by gdoffe — CHANGES_REQUESTED** [review]
Change default obstacle height to 600 to be detected **by the** Lidar.

File: cogip/sensor.py (hunk 1 above)
Comment (ecourtois) [reply cogip/sensor.py:41]: This was a test. Reverted.

File: cogip/sensor.py (hunk 2 above)
Comment (ecourtois) [reply cogip/sensor.py:59]: It's milliseconds. Reverted also.
Obstacles are detected each time the timer timeout (here every 50 ms).

**Review by gdoffe — APPROVED** [review]

**Pull request merged**